misc-bench/cpp-copy/baselines/CPU/copy.cpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file copy.cpp
+ * @brief Template for a Host Application Source File.
+ *
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <getopt.h>
+#include <assert.h>
+#include <stdint.h>
+#include <iomanip>
+#include <chrono>
+#include <omp.h>
+#include <iostream>
+#include <vector>
+
+#include "utilBaselines.h"
```

Comment:
The util file is in the misc-bench directory. So I assume this include will not work?

Reply:
Yes, as I have included the directory in the include path in the Makefile. I can reformat it to be like the others if you would like though.

Reply:
This is fine, thank you!